src/app/app.permissions.js

```diff
@@ -0,0 +1,14 @@
+angular.module('librarium').run(function (PermRoleStore, Session, UserPersistence) {
+  // logged in users
+  PermRoleStore.defineRole('USER', []);
+  // logged out users
+  PermRoleStore.defineRole('ANONYMOUS', function () {
+    const sessionStr = UserPersistence.getCookieData();
```

Comment:
This logic shouldn't be here, create a Session service that holds the logged user, it should read the user from memory if no user is currently set then read it from local storage, not cookies